src/scss/footer/_footer.scss

```diff
@@ -0,0 +1,160 @@
+.footer {
+  background-color: $footerGray;
+
+  &-toggle-list {
+    border-bottom: 1px solid $footerDarkGrey;
+    font-weight: bold;
+    line-height: 30px;
+    padding: 20px 23px;
+  }
+}
+
+.footer-category {
```

Comment:
If this not in the scope of "footer" BEM block it should be in separated scss module. Please use context and don't break out of the scope.